instrumentation/cassandra/src/main/java/brave/cassandra/CassandraTraceKeys.java

```diff
@@ -0,0 +1,11 @@
+package brave.cassandra;
+
+public final class CassandraTraceKeys {
```

Comment:
it's interesting that you use `final class` here, as appropriate, yet not on the other class.
i would not presume that the other classes are designed for being subclassed (which are in fact broadening the API being published beyond what you're probably intending it to be.)